src/OrchardCore.Modules/OrchardCore.Rules/Drivers/ContentTypeConditionEvaluatorDriver.cs

```diff
@@ -15,13 +15,14 @@ namespace OrchardCore.Rules.Drivers
     /// </summary>
     public class ContentTypeConditionEvaluatorDriver : ContentDisplayDriver, IConditionEvaluator
     {
-        private static ValueTask<bool> True => new ValueTask<bool>(true);
-        private static ValueTask<bool> False => new ValueTask<bool>(false);
+        private static readonly ValueTask<bool> _true = new(true);
```

Comment:
Do not make this change to =

Value task must create a new instance every time, ie =>

Reply:
You could actually replace this with ValueTask.FromResult(true) etc as that’s been added to the framework since this code was written

Reply:
@deanmarcussen done. Please approve if you have no further changes